classes/class-wc-connect-settings-pages.php

```diff
@@ -89,9 +89,25 @@ public function output_shipping_settings_screen() {
 				$extra_args['order_id'] = $_GET['from_order'];
 				$extra_args['order_href'] = get_edit_post_link( $_GET['from_order'] );
 			}
-			if ( isset( $_GET[ 'carrier' ] ) ) {
-				$extra_args['carrier'] = $_GET[ 'carrier' ];
+			if ( isset( $_GET['carrier'] ) ) {
+				$extra_args['carrier']    = $_GET['carrier'];
 				$extra_args['continents'] = $this->continents->get();
+				$translated_carrier_name  = null;
+				switch ( strtolower( $extra_args['carrier'] ) ) {
+					case 'ups':
+						$translated_carrier_name = __( 'UPS', 'woocommerce-services' );
+						break;
+					default:
+						break;
+				}
+				if ( $translated_carrier_name ) {
+				?>
+					<h2>
+						<a href="<?php echo esc_url( admin_url( 'admin.php?page=wc-settings&tab=shipping&section=woocommerce-services-settings' ) ); ?>"><?php esc_html_e( 'WooCommerce Services', 'woocommerce-services' ); ?></a> &gt;
+						<span class="wc-shipping-zone-name"><?php echo esc_html( $translated_carrier_name ); ?></span>
```

Comment:
(minor) I don't see a style assigned to `wc-shipping-zone-name` - is that going to be done in the future? Or can it be removed?

Reply:
You're right, that class name shouldn't be there! I'll update the PR before merging it. Thanks for the review!! :bow: